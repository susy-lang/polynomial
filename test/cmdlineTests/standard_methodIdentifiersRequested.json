{
	"language": "Polynomial",
	"sources":
	{
		"A":
		{
			"content": "pragma polynomial >=0.0; contract C { function f() public pure {} }"
		}
	},
	"settings":
	{
		"outputSelection":
		{
			"*": { "*": ["svm.methodIdentifiers"] }
		}
	}
}
