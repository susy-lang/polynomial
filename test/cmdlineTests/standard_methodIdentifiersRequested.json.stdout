{"contracts":{"A":{"C":{"svm":{"methodIdentifiers":{"f()":"26121ff0"}}}}},"sources":{"A":{"id":0}}}
